src/Microsoft.Health.Dicom.Client/Models/PartitionEntry.cs

```diff
@@ -4,22 +4,15 @@
 // -------------------------------------------------------------------------------------------------
 
 using System;
-using EnsureThat;
 
 namespace Microsoft.Health.Dicom.Client.Models;
 
-public class PartitionEntry
+/// <summary>
+/// Used to ensure API contract does not change while we change the underlying property naming.
+/// </summary>
+public class PartitionEntry : Partition
```

Comment:
What's the point of this class now? I don't think anything operates on this type, nor does the type name matter here because we only cared about it for Swagger (even though that's not such a big deal either lol)